config/nrfconnect/chip-module/Kconfig

```diff
@@ -284,4 +282,16 @@ config CHIP_ENABLE_READ_CLIENT
 	  This config can be disabled for device types that do not require Read Client functionality.
 	  Disabling this config can save flash and RAM space.
 
+config CHIP_CRYPTO_PSA_MIGRATE_DAC_PRIV_KEY
+	bool "Migrate DAC private key from factory data to PSA ITS"
+	default y
```

Comment:
This should be `y` only if ITS is encrypted (either TF-M or ITS with HUK is used - not sure what's Kconfig for the latter).

Reply:
Ok, so `CONFIG_BUILD_WITH_TFM` or `CONFIG_SECURE_STORAGE_BACKEND_AEAD_KEY_DERIVE_FROM_HUK`